core-detekt/src/main/kotlin/com/twitter/rules/core/detekt/TwitterDetektRule.kt

```diff
@@ -58,11 +58,13 @@ abstract class TwitterDetektRule(
 
     override fun visitClass(klass: KtClass) {
         super<Rule>.visitClass(klass)
+        klass.attach(config)
         visitClass(klass, autoCorrect, emitter)
     }
 
     override fun visitKtElement(element: KtElement) {
         super.visitKtElement(element)
+        element.attach(config)
```

Comment:
I found an issue in which visitKtElement was being called while visitFile wasn't 🤷 so I am attaching the config in all entrypoints, and made `attach` to be fine with that.